### PR TITLE
[BUGFIX] Ignore document nodes with metaRobotsNofollow set to TRUE

### PR DESCRIPTION
**Pull request by urbantrout**

### CODE REVIEW

**Comment by aertmann** [comment]
Hey, glad to see you figured it out. Thanks for pushing a fix, will add some suggestions for improvements so it can be accepted into the package.

File: Resources/Private/Templates/Page/XmlSiteMap.xml [hunk 1]
```diff
@@ -6,10 +6,13 @@
...
+		    	<loc>{neos:uri.node(node: item.node, format: 'html', absolute: true)}</loc>
+		    	<f:if condition="{item.node.properties.xmlSitemapChangeFrequency}"><changefreq>{item.node.properties.xmlSitemapChangeFrequency}</changefreq></f:if>
+		    	<priority>{f:if(condition: item.node.properties.xmlSitemapPriority, then: item.node.properties.xmlSitemapPriority, else: '0.5')}</priority>
+		    </url>
+        </f:if>
+        <f:if condition="{item.subItems}"><f:render section="itemsList" arguments="{items: item.subItems}"/></f:if>
 	</f:for>
-</f:section>
\ No newline at end of file
+</f:section>
```
Comment (aertmann) [inline Resources/Private/Templates/Page/XmlSiteMap.xml]: the indentation is broken with this change, before it was indented with tabs whereas now it's mixed tabs and spaces in no clear order

**Comment by aertmann** [comment]
Also this PR should be made against the lowest maintained branch (1.0) – read more on https://discuss.neos.io/t/creating-a-pull-request/506

To do that you need to close the PR and open a new one, but please check the inline comment first

File: Resources/Private/Templates/Page/XmlSiteMap.xml [hunk 2]
```diff
@@ -6,10 +6,13 @@
 </urlset>
 <f:section name="itemsList">
 	<f:for each="{items}" as="item">
-		<url>
-			<loc>{neos:uri.node(node: item.node, format: 'html', absolute: true)}</loc>
-			<f:if condition="{item.node.properties.xmlSitemapChangeFrequency}"><changefreq>{item.node.properties.xmlSitemapChangeFrequency}</changefreq></f:if>
-			<priority>{f:if(condition: item.node.properties.xmlSitemapPriority, then: item.node.properties.xmlSitemapPriority, else: '0.5')}</priority>
-		</url><f:if condition="{item.subItems}"><f:render section="itemsList" arguments="{items: item.subItems}"/></f:if>
+        <f:if condition="{item.node.properties.metaRobotsNoindex} != 'FALSE'">
```
Comment (aertmann) [inline Resources/Private/Templates/Page/XmlSiteMap.xml]: it's a bit tricky to read, maybe ``<f:if condition="{item.node.properties.metaRobotsNoindex}"><f:else>`` would be simpler?

**Comment by urbantrout** [comment]
fixed the intendation and condition

**Comment by aertmann** [comment]
Thanks, but did you see my last comment? Should be a couple of clicks.

**Comment by urbantrout** [comment]
I did. I am just not at home right now. But I'll do that when I get back
home later.

Aske Ertmann <notifications@github.com> schrieb am Mo., 26. Okt. 2015 17:29:

> Thanks, but did you see my last comment? Should be a couple of clicks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neos/neos-seo/pull/3#issuecomment-151197421>.
>
-- 
Harald Atteneder
*UT urban trout Software e.U.*
Hubergasse 14/17-19
1160 Wien - Austria

**Pull request closed without merging**